content/content-tool/tool/src/webapp/vm/content/sakai_filepicker_select.vm

```diff
@@ -242,7 +262,7 @@ $(document).ready(function() {
 						class="even#if($item.isSelectedForCopy()) selectedCopy#end#if($item.isSelectedForMove()) selectedMove#end#if(!$item.isAvailable()) inactive#end"
 					#end>
 					######################  Item title and action links  ##############
-					<td headers="title" style="text-indent:$width">
+					<td headers="title" style="text-indent:$width" class="xxx">
```

Comment:
I'm not sure this change should be in here, there's no reference to the xxx class.